iroh-ctl/run.rs

```diff
@@ -0,0 +1,27 @@
+async fn main_cli(cli: Cli) -> Result<()> {
```

Comment:
why is this not in the src dir?

Reply:
I don't know how I managed to do it, but that file isn't supposed to exist. I'll remove it.